.github/workflows/python-app.yml

```diff
@@ -0,0 +1,74 @@
+name: Build Status
+   
+on:
+   push:
+   pull_request:
+   schedule:
+     - cron: '0 0 * * 0' # weekly
+
+defaults:
+  run:
+    shell: bash
```

Comment:
Is `bash` not the default? why do you need `bash` specifically, please add a comment.